index.html

```diff
@@ -0,0 +1,220 @@
+        <img class="red-line" src="./images/red-line.png" alt="red-line" />
+      </div>
+      <div class="program-container">
+        <div></div>
+        <div class="program-card">
+          <img
+            id="write-icon"
+            class="write-icon"
+            src="./images/write-icon.svg"
+            alt="writing-icon"
+          />
+          <h3 class="program-title">Write</h3>
+          <p class="program-description">
+            How to write from personal experience and develop anecdote and
+            memory into a piece of fiction.
+          </p>
+        </div>
+        <div class="program-card">
+          <img
+            id="characters-icon"
+            class="characters-icon"
+            src="./images/characters-icon.svg"
+            alt="characters-icon"
+          />
+          <h3 class="program-title">Character</h3>
+          <p class="program-description">
+            How to create a complex protagonist.The difference between primary
+            and secondary characters.
+          </p>
+        </div>
+        <div class="program-card">
+          <img
+            id="story-icon"
+            class="story-icon"
+            src="./images/story-icon.svg"
+            alt="story-icon"
+          />
+          <h3 class="program-title">Story</h3>
+          <p class="program-description">
+            Thematic development in story. Planning and execution in story. How
+            to identify the themes in self-created writing
+          </p>
+        </div>
+        <div class="program-card">
+          <img
+            id="workshop-icon"
+            class="workshop-icon"
+            src="./images/workshop-icon.svg"
+            alt="workshop-icon"
+          />
+          <h3 class="program-title">Workshop</h3>
+          <p class="program-description">
+            Apply what you learned. Start creating your story and plan it for
+            brief presentation to peers and professors.
+          </p>
+        </div>
+        <div class="program-card">
+          <img
+            id="network-icon"
+            class="network-icon"
+            src="./images/network-icon.svg"
+            alt="network-icon"
+          />
+          <h3 class="program-title">Present</h3>
+          <p class="program-description">
+            Present your work and get professional feedback from top tier
+            professors. Network with your peers and professors.
+          </p>
+        </div>
+        <div></div>
+      </div>
+      <div class="button-wrapper">
+        <button class="join-button">Join Cohort Course 2022</button>
+      </div>
+
+      <a href="#" class="program-link">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section class="featured-speakers">
+      <h2 class="speakers">Featured Speakers</h2>
+      <div class="red-line-container">
+        <img class="red-line" src="./images/red-line.png" alt="red-line" />
+      </div>
+      <div class="speakers-container"></div>
+    </section>
```

Comment:
- [ ] It is required to produce a close replica of the of the guiding template. This includes, font-size, color, margin, padding among others. This part of the project seems to lack some features of the guiding template. Kindly look at the areas I marked with marker and compare your design with that of the given template and work your project along that direction. 
- [ ] Also, there is a dot-square under every individual speaker in the guiding template which is not reflecting on your project. Kindly look into that too and fix it.

###  Your Design

![your-speakers-section](https://user-images.githubusercontent.com/26597432/196835862-0791753f-fbe9-4e83-87a9-0b9f9480a143.png)


### Guiding Template

![temp-speaker-section](https://user-images.githubusercontent.com/26597432/196835929-b8c3e733-9a34-4d12-83cf-1bb2822f8914.png)